### DISCUSSION TITLE
Front End

### DISCUSSION THREAD

**Discussion by rizqyn9** [upkoding/upkoding#32] [Ideas]
Bang kalau boleh saran project FE sama BE nya di split, mungkin gunain framework lik React / Vue, mungkin bakal banyak contributors yang bakal join. Dari communityy ku juga mau ikut bantu, tapi karena monolith jadi susah dibuat project management. which klo  pakai framework kita bisa bagi as Component. sekedar saran sih bang :) communiy ku pengen contribs soalnya. Thanks and good luck 👍

**Comment by ekaputra07** [comment]
Halo @rizqyn9 terima kasih atas masukkannya, dari awal sebenarnya saya sudah ada pertimbangan untuk dipisah tapi waktu itu saya belum tahu kemana arahnya proyek ini. Si stage awal dimana kita masih meraba2 apa yang pengguna inginkan maka saya ingin yang bisa develop dengan cepat tanpa kompleksitas yang belum diperlukan jadi saya putuskan monolith dulu, meskipun peluang untuk displit selalu terbuka.

Tapi meski monolith, template dan JS sudah terpisah jadi development sebenarnya tidak harus tahu Django atau Python, tapi untuk development local memang perlu jalanin dua-duanya yaitu server django dan server UI nya (gulp), tapi itu juga tidak terlalu sulit karena sudah pakai Docker compose.

Cara lain untuk kontribusi juga bisa dengan bikin challenge :) let me know if you interested 🙏
